hilo.rb

```diff
@@ -0,0 +1,26 @@
+random_number = rand(1..100)
```

Comment:
Pardon me for the nitpick, but it looks like your main branch is empty. You might want to merge an initial version.  Also, you can assign me a a reviewer once you are ready for a review.